src/me/ryanhamshire/GriefPrevention/command/CommandGpBlockInfo.java

```diff
@@ -0,0 +1,45 @@
+package me.ryanhamshire.GriefPrevention.command;
+
+import me.ryanhamshire.GriefPrevention.GriefPrevention;
+import org.spongepowered.api.command.CommandException;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.spec.CommandExecutor;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.text.Texts;
+
+public class CommandGpBlockInfo implements CommandExecutor {
+
+    @Override
+    public CommandResult execute(CommandSource src, CommandContext ctx) {
+        Player player;
+
+        try {
+            player = GriefPrevention.instance.checkPlayer(src);
```

Comment:
Also checkPlayer should probably be static so you don't need to mess with the plugin instance here.